NuGet.Config

```diff
@@ -23,6 +23,7 @@
     <add key="myget.org dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <add key="myget.org symreader" value="https://dotnet.myget.org/F/symreader/api/v3/index.json" />
     <add key="myget.org symreader-portable" value="https://dotnet.myget.org/F/symreader-portable/api/v3/index.json" />
+    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
```

Comment:
Rename to `myget.org roslyn-master-nightly` to match the naming schemes of our other sources.